monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -225,6 +226,7 @@ def get_island_cross_segment_issues(cls):
             if not found_good_ip:
                 issues.append(
                     {
+                        "id": island_machine.id,
                         "type": "island_cross_segment",
                         "machine": island_machine.hostname,
```

Comment:
We probably no longer need this, because if we have `machine_id` then we can pull its hostname anytime. But I imagine this would cause a bigger refactoring

Reply:
It would and considering that we're going to rework this anyway, I'd rather not touch it.